home-work-angular/app-cat/directives/ca.imagepreview.directive.js

```diff
@@ -0,0 +1,17 @@
+(function (module) {
+    'use strict';
+
+
+    function imagePreview() {
+        return {
+            restrict: 'EA',
+            scope: {
+                catUrl: '=catUrl',
```

Comment:
Scope property name and value are same with just '=' as difference, you can use just '=' in this case.